ironfish/src/consensus/verifier.ts

```diff
@@ -327,22 +331,21 @@ export class Verifier {
    * spend's spend root.
    *
    * @param spend the spend to be verified
-   * @param size the size of the nullifiers tree at which the spend must not exist
```

Comment:
This should probably be added back, or we should kill the whole comment so it's at least somewhat consistent

Reply:
Fixed